### PR TITLE
Updates to work with new helper mode banner

### PR DESCRIPTION
**Pull request by hmsommer**
changes:
- If HF2019 & helper mode - centers dialog in space that excludes the banner
- Adjusts heigh of space allowed for helper banner
- Only gives space for helper banner if HF is showing helper mode and the user is in helper mode.

For this story: https://www5.v1host.com/FH-V1/story.mvc/Summary?oidToken=Story%3A1082918

### CODE REVIEW

**Comment by justincy** [comment]
Does https://github.com/fs-webdev/hf/pull/901 have to be merged and released first since it adds the `HF.showHelper` var?

**Comment by hmsommer** [comment]
@justincy Either way it’s going to be a little buggy with the gap or lack of gap (depending on which one is merged first) above the dialogs, but it’s all under that 2019 hf experiment, so guessing it doesn’t really matter which one goes first. I hope to merge them around the same time though.

**Comment by rosemariesadler** [comment]
could you post some screenshots of what this looks like both on a mobile view and desktop view?

**Comment by hmsommer** [comment]
@rosemariesadler 
Desktop:
image
mobile:
image

File: fs-dialog-base.html [hunk 1]
```diff
@@ -334,8 +340,8 @@
 
         fs-modal-dialog[role-indicator-on]:not([no-fullscreen-mobile]),
         fs-anchored-dialog[role-indicator-on]:not([no-fullscreen-mobile]) {
-          height: calc(100% - 40px) !important;
-          margin-top: 40px;
+          height: calc(100% - 46px) !important;
```
Comment (rosemariesadler) [inline fs-dialog-base.html]: will this make things look wrong without the hf2019 experiment on?

Comment (hmsommer) [reply fs-dialog-base.html]: We recently made changes to the old helper banner to be full width on mobile. The new 2019 helper banner is always full width. However, the new helper banner was a few px taller than the older helper banner, so I adjusted the old helper banner to be the same height as the new one. The changes here are adjustments for that change in height.

**Pull request merged**